### PR TITLE
Implement active layers counters for entry boxes

### PR DESCRIPTION
**Pull request by kfilip94**
This PR adds counters to entry boxes:
[PIVOTAL](https://www.pivotaltracker.com/story/show/166617858)

Screenshot from 2019-06-12 12 51 13

### CODE REVIEW

File: src/components/entry-boxes/entry-boxes.js [hunk 1]
```diff
@@ -1,13 +1,12 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import Component from './entry-boxes-component';
-
+import { getCountedActiveLayers } from './entry-boxes-selectors';
```
Comment (weberjavi) [inline src/components/entry-boxes/entry-boxes.js]: @aabdaab , @kfilip94 just opening this conversation.
I prefer the pattern we where doing lately of exporting `default` from selectors files an structured selector (`createStructuredSelector({ ...selectors map in here... }))`) an doing `import mapStateToProps from './selectors'` than importing the named selectors and creating the `mapStateToProps` on the entry file. I think the component container file it is cleaner and having the structured selector at the end of the selectors file also looks quite terse.
What do you think?

Comment (aabdaab) [reply src/components/entry-boxes/entry-boxes.js]: I also prefer this way. If you create a `selectors` file anyway you might just as well export the `createStructuredSelector` there; this way it's a lot easier to add more props to the component without lots of changes

Comment (kfilip94) [reply src/components/entry-boxes/entry-boxes.js]: I agree with you, it can be a cleaner approach. I'm going to change it now :+1:

**Comment by kfilip94** [comment]
@weberjavi, @aabdaab ready again :point_up: :point_up_2:

**Pull request merged**